sqlitecluster/SQLiteNode.cpp

```diff
@@ -1949,7 +1949,7 @@ void SQLiteNode::_changeState(SQLiteNode::State newState) {
     SQLiteNode::State oldState = _state;
     if (newState != oldState) {
         // Depending on the state, set a timeout
-        SDEBUG("Switching from '" << stateNames[_state] << "' to '" << stateNames[newState] << "'");
+        SINFO("Switching from '" << stateNames[_state] << "' to '" << stateNames[newState] << "'");
```

Comment:
did you run tests to ensure that this is not going to be super noisy? Should be easy to run the test suite and just tail the logs for this line.

Reply:
Tailed the logs while I had a 5 node cluster up and was doing ~1 new expense per second, 100 times. Not a single log line:
```
vagrant@expensidev:/vagrant/Bedrock$ tail -f /var/log/syslog | grep "Switching from"





```
On the same cluster, did a standup of two master-eligible nodes that needed to sync, and it looked like this:
```
vagrant@expensidev:/vagrant/Bedrock$ sudo service auth1 start; sudo service auth start; tail -f /var/log/syslog | grep "Switching from"
2019-04-05T19:55:46.558027+00:00 expensidev auth1:  (SQLiteNode.cpp:1961) _changeState [sync] [info] {auth1/SEARCHING} Switching from 'SEARCHING' to 'SYNCHRONIZING'
2019-04-05T19:55:46.854495+00:00 expensidev bedrock:  (SQLiteNode.cpp:1961) _changeState [sync] [info] {auth/SEARCHING} Switching from 'SEARCHING' to 'SYNCHRONIZING'
2019-04-05T19:55:56.956067+00:00 expensidev auth1: xxxxxx (SQLiteNode.cpp:1961) _changeState [sync] [info] {auth1/SYNCHRONIZING} Switching from 'SYNCHRONIZING' to 'WAITING'
2019-04-05T19:55:56.959159+00:00 expensidev auth1:  (SQLiteNode.cpp:1961) _changeState [sync] [info] {auth1/WAITING} Switching from 'WAITING' to 'SUBSCRIBING'
2019-04-05T19:55:56.960143+00:00 expensidev auth2:  (SQLiteNode.cpp:1961) _changeState [sync] [info] {auth2/MASTERING} Switching from 'MASTERING' to 'STANDINGDOWN'
2019-04-05T19:55:56.962315+00:00 expensidev auth2:  (SQLiteNode.cpp:1961) _changeState [sync] [info] {auth2/STANDINGDOWN} Switching from 'STANDINGDOWN' to 'SEARCHING'
2019-04-05T19:55:56.964481+00:00 expensidev auth2:  (SQLiteNode.cpp:1961) _changeState [sync] [info] {auth2/SEARCHING} Switching from 'SEARCHING' to 'WAITING'
2019-04-05T19:55:56.971053+00:00 expensidev auth4:  (SQLiteNode.cpp:1961) _changeState [sync] [info] {auth4/SLAVING} Switching from 'SLAVING' to 'SEARCHING'
2019-04-05T19:55:56.974690+00:00 expensidev auth4:  (SQLiteNode.cpp:1961) _changeState [sync] [info] {auth4/SEARCHING} Switching from 'SEARCHING' to 'WAITING'
2019-04-05T19:55:56.979116+00:00 expensidev auth1: xxxxxx (SQLiteNode.cpp:1961) _changeState [sync] [info] {auth1/SUBSCRIBING} Switching from 'SUBSCRIBING' to 'SEARCHING'
2019-04-05T19:55:56.983270+00:00 expensidev auth3:  (SQLiteNode.cpp:1961) _changeState [sync] [info] {auth3/SLAVING} Switching from 'SLAVING' to 'SEARCHING'
2019-04-05T19:55:56.983758+00:00 expensidev auth1:  (SQLiteNode.cpp:1961) _changeState [sync] [info] {auth1/SEARCHING} Switching from 'SEARCHING' to 'WAITING'
2019-04-05T19:55:56.986467+00:00 expensidev auth3:  (SQLiteNode.cpp:1961) _changeState [sync] [info] {auth3/SEARCHING} Switching from 'SEARCHING' to 'WAITING'
2019-04-05T19:56:01.874692+00:00 expensidev bedrock: xxxxxx (SQLiteNode.cpp:1961) _changeState [sync] [info] {auth/SYNCHRONIZING} Switching from 'SYNCHRONIZING' to 'WAITING'
2019-04-05T19:56:01.876508+00:00 expensidev bedrock:  (SQLiteNode.cpp:1961) _changeState [sync] [info] {auth/WAITING} Switching from 'WAITING' to 'STANDINGUP'
2019-04-05T19:56:01.920664+00:00 expensidev bedrock:  (SQLiteNode.cpp:1961) _changeState [sync] [info] {auth/STANDINGUP} Switching from 'STANDINGUP' to 'MASTERING'
2019-04-05T19:56:01.927450+00:00 expensidev auth4:  (SQLiteNode.cpp:1961) _changeState [sync] [info] {auth4/WAITING} Switching from 'WAITING' to 'SUBSCRIBING'
2019-04-05T19:56:01.928555+00:00 expensidev auth2:  (SQLiteNode.cpp:1961) _changeState [sync] [info] {auth2/WAITING} Switching from 'WAITING' to 'SUBSCRIBING'
2019-04-05T19:56:01.930804+00:00 expensidev auth1:  (SQLiteNode.cpp:1961) _changeState [sync] [info] {auth1/WAITING} Switching from 'WAITING' to 'SUBSCRIBING'
2019-04-05T19:56:01.933639+00:00 expensidev auth3:  (SQLiteNode.cpp:1961) _changeState [sync] [info] {auth3/WAITING} Switching from 'WAITING' to 'SUBSCRIBING'
2019-04-05T19:56:01.956259+00:00 expensidev auth1: xxxxxx (SQLiteNode.cpp:1961) _changeState [sync] [info] {auth1/SUBSCRIBING} Switching from 'SUBSCRIBING' to 'SLAVING'
2019-04-05T19:56:01.963764+00:00 expensidev auth2: xxxxxx (SQLiteNode.cpp:1961) _changeState [sync] [info] {auth2/SUBSCRIBING} Switching from 'SUBSCRIBING' to 'SLAVING'
2019-04-05T19:56:01.969664+00:00 expensidev auth3: xxxxxx (SQLiteNode.cpp:1961) _changeState [sync] [info] {auth3/SUBSCRIBING} Switching from 'SUBSCRIBING' to 'SLAVING'
2019-04-05T19:56:01.982435+00:00 expensidev auth4: xxxxxx (SQLiteNode.cpp:1961) _changeState [sync] [info] {auth4/SUBSCRIBING} Switching from 'SUBSCRIBING' to 'SLAVING'



```
(the empty lines at the end are where it stopped logging the state change line)

Reply:
It is an inconsequential amount of logs, that really only show up when the cluster state changes, e.g. we add or remove a master, which is when these changes are helpful for establishing the precise timing of state changes.